### DISCUSSION TITLE
There really needs to be a better form of moderation when it comes to posts in the community discussions - such as report a post?

### DISCUSSION THREAD

**Discussion by chris-c-thomas** [community/community#46397] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

Low quality posts are one thing. They don't really violate anything and can be distracting but people answer and are generally great. I'm sure I've made plenty of those across message boards myself.

But then there's just random things like: https://github.com/community/community/discussions/46385 that stay up for hours that are just self promotion, spam, random sentences put together like it's ChatGPT's first day on the job...

Even something as basic as reporting a post, something that exists almost everywhere, would be a good first start.

**Comment by Akash1134** [comment, marked as answer]
Hi @chris-c-thomas , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
